nixos/modules/services/audio/mpd.nix

```diff
@@ -181,7 +193,12 @@ in {
 
       serviceConfig = {
         User = "${cfg.user}";
-        ExecStart = "${pkgs.mpd}/bin/mpd --no-daemon ${mpdConf}";
+        ExecStart = "${pkgs.mpd}/bin/mpd --no-daemon /etc/mpd.conf";
+        ExecStartPre = pkgs.writeScript "mpd-start-pre" ''
```

Comment:
Maybe we should only do this when `credentialsFile` is actually used?

Reply:
The permissions are one thing and the (not strictly necessary) extra file (potentially overwriting an existing file) is another. But I guess since `/etc/mpd.conf` would be written in some cases anyway it's actually simpler/cleaner to keep it as is.